docs/content/templates/functions.md

```diff
@@ -155,14 +155,17 @@ e.g.
 
 ### after
 Slices an array to only the items after the <em>N</em>th item. Use this in combination
-with `first` to use both halves of an array split at item _N_.
+with `first` to use both (not necessarily equal) halves of an array split at item _N_.
 
 Works on [lists](/templates/list/), [taxonomies](/taxonomies/displaying/), [terms](/templates/terms/), [groups](/templates/list/)
 
 e.g.
 
-    {{ range after 10 .Data.Pages }}
-        {{ .Render "title" }}
+    {{ range after 2 .Data.Pages }}
```

Comment:
I don't know if adding an example of `first` is necessary here.  Also, I think it may help the illustration if we use a simple slice in the example instead of `.Data.Pages`.

```
{{ $data := slice "one" "two" "three" "four" }}

{{ range after 2 $data }} → ["three", "four"]
{{ . }}
{{ end }}
```